fieldmanager.php

```diff
@@ -499,6 +499,23 @@ function fm_sanitize_textarea( $value ) {
 }
 
 /**
+ * Call wp_die() with an error message for Fieldmanager_Field::_failed_validation().
+ *
+ * @since 1.1.0
+ *
+ * @param string $error_message Optional. Error message in addition to the
+ *                              default message for validation errors.
+ */
+function fm_failed_validation_to_wp_die( $error_message = '' ) {
+	wp_die( esc_html(
+		$error_message
+		. "\n\n"
+		. __( "You may be able to use your browser's back button to resolve this error.", 'fieldmanager' )
+	) );
+}
+add_action( 'fm_failed_validation', 'fm_failed_validation_to_wp_die' );
```

Comment:
All of the arguments against `wp_die()` seem convincing to me.

I think I get the relative advantages of option (2), but I want to make sure I understand it correctly. Would these points need to be addressed as part of implementing it?

- We would need some way of catching a validation error in field that is a child of a `Fieldmanager_Group` and blocking saving for the rest of the group (except for fields with `serialize_data => false`?).

- We would need to make sure we had a way to display the admin notice for fields that save over Ajax, like Quick Edit.

- Arguably, we would need a plan for displaying notices for contexts outside of the Dashboard. Although no such contexts currently exist in FM core (I don't thnk), the proposed user-forms (#157) and Customizer (#399) contexts come to mind.